repose-aggregator/src/docs/asciidoc/filters/attribute-mapping-policy-validation.adoc

```diff
@@ -17,7 +17,7 @@ This filter has no configuration.
 
 == Prerequisites & Postconditions
 === Required Request Headers
-* `Content-Type` - Declares the content type of the policy (e.g., XML or JSON) which will be used during validation.
+* `Content-Type` - Declares the content type of the policy, must be YAML.
```

Comment:
Technically, this filter will process any `Content-Type` that includes `yaml`. See:
https://github.com/rackerlabs/repose/pull/1770/files#diff-90c6141eddf308aab0b584febbfd0308R96